imgui-binding/build.gradle

```diff
@@ -34,3 +35,19 @@ jar {
         attributes  'Automatic-Module-Name': 'imgui.binding'
     }
 }
+
+tasks.register('makePropertyResource') {
+    def directory = file "src/main/resources/META-INF"
+    Files.createDirectories(directory.toPath())
+    def propertyFile = file "${project.projectDir}/src/main/resources/META-INF/imgui-java.properties"
+    def props = new Properties()
+    if (propertyFile.exists()) {
+        propertyFile.withReader { props.load(it) }
+    }
+    props.setProperty("version", project.version)
+    propertyFile.withWriter { props.store(it, "imgui-java") }
+}
+
+processResources {
+    dependsOn makePropertyResource
```

Comment:
I recommend to do the next:

1. Generate properties file to the `buildDir`.
2. Change `processResources` like:

```
processResources {
    dependsOn makePropertyResource
    from "$buildDir/path-to-file/imgui-java.properties"
}
```

Thus you can avoid creating of unrequired `.gitignore`.

Reply:
`from "$buildDir/path-to-file/imgui-java.properties"` is not needed if I place it in buildDir/resources/main.